app/src/test/java/io/intrepid/contest/screens/contestoverview/ContestOverviewPresenterTest.java

```diff
@@ -58,6 +58,36 @@ public void onViewCreatedShouldShowContestNameWhenSuccessfullyRetrievedContestDe
     }
 
     @Test
+    public void onViewCreatedShouldShowRatingGuide() throws Exception {
+        setupSuccessfulContestDetailsCall();
+
+        presenter.onViewCreated();
+        testConfiguration.triggerRxSchedulers();
+
+        verify(mockView).showRatingGuide();
+    }
+
+    @Test
+    public void onViewCreatedShouldShowContestDescriptionWhenSuccessfullyRetrievedContestDetails() throws Exception {
```

Comment:
We kind of created a pattern of `onEventShouldDoSomethingWhenSomething` if there is the need for an event to be fired (not right after presenter constructor) and if there is any condition (in this case it shouldn't happen if the contest details call fails)...
I get your point though, so it's up to abass